makeflow/src/makeflow.c

```diff
@@ -177,6 +178,74 @@ static struct list *shared_fs_list = NULL;
 
 static int did_find_archived_job = 0;
 
+static struct list *jx_argv = NULL;
+static struct jx *jx_optarg = NULL;
+
+static void push_jx_argv(struct jx *j) {
+	assert(j);
+	if (!jx_argv)
```

Comment:
Please use braces, even for single statement ifs. We had problems in the past where what looked like a single statement was a macro expansion, and that was fun to debug.